### ISSUE TITLE
[web-profiler-bundle] Peak memory usage is always 0.00Mb.

### ISSUE THREAD

**Issue by naroga** [symfony/symfony#16836]
Since I upgraded to Symfony 2.8 and PHP 7.0, my Web Profiler shows that my application always has a peak memory usage of 0.00mb.

Screenshot attached. Please notice the page provided dispatches 6 queries, all using doctrine's object hydration. There is no way my peak memory usage went down from 5~8mb to 0.00mb.

Ubuntu 15.10, PHP 7.0 (stable), Symfony 2.8.

![screenshot from 2015-12-04 12-17-06](https://cloud.githubusercontent.com/assets/3511149/11591751/6129b9d6-9a81-11e5-9994-75c1e9c4ccac.png)

**Comment by Tobion** [comment]
PHP 7 is so good it doesn't need memory at all ;)

**Comment by SenseException** [comment]
I installed a default Symfony 2.8 and PHP 7 (Ubuntu 14.04) but cannot recreate this behaviour. Since your having an custom application running it is possible, that the DataCollector for memory is losing its value during the request/response for some reason.

Can you debug MemoryDataCollector in the HttpKernel component to see if this happens on the PHP function call of memory_get_peak_usage?

**Comment by SoboLAN** [comment]
@naroga Can you install the stock un-modified Symfony Standard edition and see if it reproduces there ? If not, then it could be some problem that is specific only to your application.

**Comment by Tobion** [comment]
I cannot reproduce this either.

**Comment by xabbuh** [comment]
@naroga Can you provide some steps to reproduce your behaviour?

**Comment by jakzal** [comment]
@naroga is your app running on NFS (see https://github.com/symfony/symfony/issues/17030#issuecomment-165732318)?

**Comment by naroga** [comment]
I had to format my partitions, and I took the opportunity to install a new default symfony 2.8. I could not reproduce this error either.

However, my previous project still has this issue. It happened when using the recipe to upgrade from 2.7 to 2.8 ([here](http://symfony.com/doc/current/cookbook/upgrade/minor_version.html)). It worked just fine on Symfony 2.7.

Sorry I can't be more helpful, I have no idea how to reproduce this again. It might be some particularity in my project, but I can't really tell.

To whoever asked, I'm now running Ubuntu 14.04, on EXT4 partitions, with nginx, PHP 7.0 FPM. Issue only happened once when I upgraded one particular project from 2.7 to 2.8. Haven't tried upgrading more, but it doesn't happen on 2.8.
